lib/heroku/kensa/check.rb

```diff
@@ -281,9 +282,11 @@ def call!
           :heroku_id => heroku_id,
           :plan => data[:plan] || 'test',
           :callback_url => callback,
+	  :log_input_url => "https://token:t.01234567-89ab-cdef-0123-456789abcdef@1.us.logplex.io/logs",
```

Comment:
@bjeanes yep, that's correct. The `log_input_url` value is only sent in the payload if the add-on service has `log_input` in its manifest: https://devcenter.heroku.com/articles/add-on-provider-log-integration#setup

What would the test case for sending `log_input_url` cover? That the provisioning service accepts the request including `log_input_url` without issue?